.golangci.yml

```
@@ -1,4 +1,6 @@
+version: "2"
 
+version: "2"
```

YAMLlint (1.38.0)

[error] duplication of key "version" in mapping

(key-duplicates)